### PR TITLE
Custom items 07/01

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds the following items:

[Mira's alchemist attire](https://forums.aurorastation.org/topic/11083-miras-alchemist-and-attire-set/)
[Ameline's Dominian Culinary Dress](https://forums.aurorastation.org/topic/10981-ameline-hemorys-dominian-culinary-attire/)
[Qrqil Qrrzix's Cane](https://forums.aurorastation.org/topic/11290-qrqils-cane-app/)
[Krin's outfit](https://forums.aurorastation.org/topic/11040-krins-punk-outfit/)
[Aavs' mask](https://forums.aurorastation.org/topic/11076-aavs-guwans-reflective-mask/)
[Paul's glasses](https://forums.aurorastation.org/topic/10800-the-only-good-thing-going-for-paul-grunnus-right-now-application/)

It also make the code better, by using to_chat and other minor changes.

Config: 
[custom_items.txt](https://github.com/Aurorastation/Aurora.3/files/2734815/custom_items.txt)

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -2471,4 +2471,196 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+	name = "dark boots"
+	desc = "A pair of black boots with tall laces."
+	icon = 'icons/obj/custom_items/mira_clothing.dmi'
+	icon_state = "mira_boots"
+	item_state = "mira_boots"
+	contained_sprite = TRUE
+
+/obj/item/clothing/under/fluff/mira_skirt //Mira's Skirt - Mira Akhandi - queenofyugoslavia
+	name = "suspended skirt"
+	desc = "A plaid skirt with suspender's, sewed into the side is the intianl \"M.A.\"."
```
Comment (ParadoxSpace) [inline code/modules/customitems/item_defines.dm]: suspenders
initial

**Comment by ParadoxSpace** [comment]
I'll allow it.

**Comment by ParadoxSpace** [comment]
pls merge

**Comment by Karolis2011** [comment]
Please understand, non bug fixes have to wait 76 hours till it's merged. It's our policy to not instantly merge backdoors / bad code.

**Pull request merged**